website/docs/feature-flag-tutorials/golang/golang-examples.md

```diff
@@ -0,0 +1,361 @@
+---
+title: Go Feature Flag Examples
+slug: /feature-flag-tutorials/golang/examples
+---
+
+In our [Go feature flag tutorial](/feature-flag-tutorials/golang), we implemented a simple feature flag that could be turned on and off. This document will walk you through some common examples of using feature flags in Go.
+
+We built many features into Unleash, our open-source feature flag platform, to address the complexities of releasing code. This tutorial will explore the following:
+
+- [Gradual Rollouts for Go Apps](#gradual-rollouts-for-go-apps)
```

Comment:
Let's use title case and drop the toc. You could still do a summary of roughly what is covered in the tutorial, but I don't think we don't need to list and link to each section